pyproject.toml

```diff
@@ -81,7 +81,8 @@ extras = [
     "aiohttp>=3.8.6",
     "watchdog>=2.3.0",
     "ruamel.yaml>=0.15.98",
-    "toml>=0.9.2",
+    "tomli;python_version<'3.11'",
+    "tomli-w",
```

Comment:
Please specify a minimum version. The one available in Debian stable or Ubuntu LTS will probably be the best option.

Reply:
Seems fairly arbitrary, I would prefer to leave it to upstream to decide on a minimum. I generally do not add bounds until I find the need for one at which point I document the need for it. It is too easy for the bounds to become out-of-sync with the actual usage in the code itself (although there are CI tricks to test the lower-bounds)

Reply:
How about setting both `>=1.0.0` then?